src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -21,7 +22,8 @@ public AuthenticationController(AuthenticationService authService,
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody UserRequestDto requestDto) {
+    public UserResponseDto register(@RequestBody
+                                        @Valid UserRequestDto requestDto) {
```

Comment:
such formatting looks unnecessary, it seems line in not so long (pls recheck all controllers)